src/components/Icons/FeCheckCircle.vue

```diff
@@ -1,10 +1,15 @@
 <template>
-  <svg width="1em" height="1em" viewBox="0 0 24 24">
-    <path
-      d="M12 22C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10s-4.477 10-10 10zM8 10l-2 2l5 5l7-7l-2-2l-5 5l-3-3z"
-      fill="currentColor"
-      fillRule="nonzero"
-    ></path>
+  <svg
+    width="24"
```

Comment:
By using `1em` as width/height you can control the size of the icon by the font size of the parent.